make_dashboards/templates/internal/js/dashboard.js

```diff
@@ -291,7 +290,6 @@ function make_balance_plot(target, aim, now, prev, subtext){
         }else{
             my_plotlines = Array();
             chroma_colors = chroma.scale(['#AF2323','#259F0B','#4c85cc']).colors(now.length);
-            pl = [];
```

Comment:
@ewels this was the reason.

Reply:
Cheeky! I wonder what he was doing there...